qemu/tests/cfg/block_stream.cfg

```diff
@@ -71,6 +71,17 @@
             snapshot_chain = "images/sn1"
             expected_image_file = "images/sn1"
             after_finished = "check_backingfile check_imagefile verify_alive"
+        - backingfile_option:
```

Comment:
Please fix the compatibility issue.
```
# @backing-file: The backing file string to write into the top
...
#                (Since 2.1)
```
And because this resaon, your patch needs to be changed accordingly.